pom.xml

```diff
@@ -312,6 +317,12 @@
       <version>1.2</version>
     </dependency>
 
+    <dependency>
+      <groupId>com.bugsnag</groupId>
+      <version>[3.0,4.0)</version>
```

Comment:
This syntax! I had to go confirm that this is in fact what bugsnag recommends, and it is.

Reply:
Yeah, I don't really understand it. I guess it's like the math syntax for range. `[]` are inclusive, `()` is exclusive